tracer/src/Datadog.Tracer.Native/debugger_probes_instrumentation_requester.cpp

```diff
@@ -180,9 +230,86 @@ void DebuggerProbesInstrumentationRequester::PerformInstrumentAllIfNeeded(const
             /* is_exact_signature_match */ false));
 
         const auto numReJITs = m_debugger_rejit_preprocessor->RequestRejitForLoadedModules(
-            std::vector{module_id}, std::vector{methodProbe},
-            /* enqueueInSameThread */ true);
-        Logger::Debug("Instrument-All: Total number of ReJIT Requested: ", numReJITs);
+            std::vector{module_id}, std::vector{methodProbe}, /* enqueueInSameThread */ true);
+
+        Logger::Debug("Instrument-All: ReJIT Requested for: ", methodProbe->target_method.method_name,
+                      ". ProbeId:", methodProbe->probeId, ". Numbers of ReJits: ", numReJITs);
+
+#ifndef _WIN32
+        return;
+#endif
```

Comment:
I would not 'cut off' the code below this way, because it will still be compiled and be part of the binary. If you want to strip out the code block below this preprocessor, surround the rest with `#else`.

But still, why do you do that?

Reply:
Fix in 9f23fdedaa2a7645d2ae2d2edbaf79936c9e57ef